index.html

```diff
@@ -15,13 +17,27 @@
 </head>
 
 <body>
-
- <!-- navigation code here -->
+  <div class="nav-bg">
+  <div class="nav">
+    <div>
```

Comment:
I feel like you could have used like one or two less divs here. But looks great. Good job.